mri/operators/fourier/orc_wrapper.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+##########################################################################
+# pySAP - Copyright (C) CEA, 2017 - 2018
+# Distributed under the terms of the CeCILL-B license, as published by
+# the CEA-CNRS-INRIA. Refer to the LICENSE file or to
+# http://www.cecill.info/licences/Licence_CeCILL-B_V1-en.html
+# for details.
+##########################################################################
+
+"""
+This modules contains classes for Off-Resonance Correction (ORC)
+
+:Author: Guillaume Daval-Frérot <guillaume.davalfrerot@cea.fr>
+"""
+
+import numpy as np
+
+from ..base import OperatorBase
+from .utils import (compute_mfi_coefficients,
+                    compute_mti_coefficients,
+                    compute_svd_coefficients,
+                    compute_fsvd_coefficients)
+
+
+class ORCFFTWrapper(OperatorBase):
+    """ Off-Resonance Correction FFT Wrapper
+
+    This class is used to wrap any FFT operator and change it
+    into an off-resonance correction multi-linear operator using
+    the method described in :cite: `sutton2003` with different
+    choices of coefficients described in :cite: `fessler2005`.
+
+    """
+    def __init__(self, fourier_op, field_map, time_vec, mask,
+                 coefficients="svd", weights="full", L="auto", n_bins="auto"):
```

Comment:
I feel its better to have something more meaningful than `L`, maybe `num_interpolators` ?